javascript/javascript2/week3/homework/fastest-presser.js

```diff
@@ -0,0 +1,73 @@
+const timerInput = document.getElementById("timer");
```

Comment:
Solution looks good and works mostly as intended, well done! It handles wins, draws, missing input and no presses.

The replay logic is a little shaky - it does not always let me replay, and it says "Play Game" and not "Play Again" when someone has won. But, those are minor things, overall well done!

Reply:
Thanks for your review!:)